dashboard/app/models/user.rb

```diff
@@ -2507,7 +2507,7 @@ def account_age_in_years
 
   # Returns a list of all grades that the teacher currently has sections for
   def grades_being_taught
-    @grades_being_taught ||= sections.map(&:grade).uniq
+    @grades_being_taught ||= sections.map(&:grades).flatten.uniq
```

Comment:
this ends up not being sorted at all, but that matches the old behavior, so I assume that's fine